packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -142,6 +141,29 @@ type ReactPriorityLevelsType = {
   NoPriority: number,
 };
 
+// https://www.npmjs.com/package/semver-compare
+function semvercmp(a: string = '', b: string = '') {
+  const pa = a.split('.');
+  const pb = b.split('.');
+  for (var i = 0; i < 3; i++) {
+    const na = Number(pa[i]);
+    const nb = Number(pb[i]);
+    if (na > nb) return 1;
+    if (nb > na) return -1;
+    if (!isNaN(na) && isNaN(nb)) return 1;
+    if (isNaN(na) && !isNaN(nb)) return -1;
+  }
+  return 0;
+}
+
+function gt(a: string = '', b: string = '') {
+  return semvercmp(a, b) === 1;
+}
+
+function gte(a: string = '', b: string = '') {
+  return semvercmp(a, b) > -1;
+}
```

Comment:
nit: These should probably go into a file like `packages/react-devtools-shared/src/backend/utils.js` since they're not specifically related to React.

Reply:
Thanks for the pointer, done!